### PR TITLE
[WIP] Changed __init__. DoraCommit modified.

### PR DESCRIPTION
**Pull request by MehtapIsik**

### CODE REVIEW

File: scripts/DoraGatherPDB.py [hunk 1]
```diff
@@ -272,7 +273,7 @@ def dictify_pdb_row(pdb_row):
     # Use multiprocessor pool to retrieve various data for each PDB
     pool = Pool()
     # results = pool.map(extract_pdb_data, [(db_pdb_row.id, current_crawl_number) for db_pdb_row in db_pdb_rows])
-    results = pool.map(extract_pdb_data, db_pdb_dicts)
+    results = map(extract_pdb_data, db_pdb_dicts) #I changed pool.map() to map() to get error messages in order -MI
```
Comment (jchodera) [inline scripts/DoraGatherPDB.py]: This will not use the `multiprocessing` pool anymore.  Are you sure you want to do this?

**Comment by danielparton** [comment]
@jchodera I'm doing a big refactor of TargetExplorer at the moment to make it testable, readable, maintainable, conda-packagable etc.
This PR from @MehtapIsik is mainly just so I can keep track of some fixes she has made in recent days. Since the refactoring is major, it is likely I will just close this PR and we'll add @MehtapIsik's changes in the origin repo.

**Comment by jchodera** [comment]
OK!

**Comment by danielparton** [comment]
We already added the changes directly into the origin repo, so we're closing this PR.

**Pull request closed without merging**